powerlevel9k.zsh-theme

```diff
@@ -1365,10 +1365,10 @@ prompt_rspec_stats() {
 ################################################################
 # Segment to display Ruby Version Manager information
 prompt_rvm() {
-  local version_and_gemset=${rvm_env_string/ruby-}
-
-  if [[ -n "$version_and_gemset" ]]; then
-    "$1_prompt_segment" "$0" "$2" "grey35" "$DEFAULT_COLOR" "$version_and_gemset" 'RUBY_ICON'
+  local rvm_prompt
```

Comment:
@HaoZeke You should probably wrap this entire thing in `if [ $commands[rvm-prompt] ]` to prevent issues if the command doesn't exist.

Like this:
```shell
prompt_rvm() {
  if [ $commands[rvm-prompt] ]; then
    local rvm_prompt
    rvm_prompt=$(rvm-prompt v p)
    if [ "$rvm_prompt" != "" ]; then
      "$1_prompt_segment" "$0" "$2" "240" "$DEFAULT_COLOR" "$rvm_prompt $(print_icon 'RUBY_ICON') "
    fi
  fi
}
```